src/components/trade/Chart/Chart.svelte

```diff
@@ -0,0 +1,114 @@
+<script>
+  import { getChainlinkPriceHistory } from "@api/markets";
+  import { onMount } from "svelte";
+  import { scaleLinear } from "d3-scale";
+
+  const leftOffset = 0;
+  const bottomOffset = 0;
+	let length;
+  let pointsY = [];
+	let height = 0, width = 0;
+	let minY
+	let maxY
+  let activeIndex = null; // hovered timestamp
+  let tooltipTimer;
+
+	onMount(async () => {
+		const dimensions = document.getElementById("chart").getBoundingClientRect();
+		height = dimensions.height
+		width = dimensions.width
+    let points = [];
+    try {
+      let priceHistory = await getChainlinkPriceHistory(
+        // "0x3607e46698d218b3a5cae44bf381475c0a5e2ca7"
+				"0x942d00008d658dbb40745bbec89a93c253f9b882"
```

Comment:
Referring to this in my main thread comment.

Reply:
> Referring to this in my main thread comment.

I left that static because I thought I would take a look at that when I created the Markets components - when the user would be able to change the market. We'll definitely store this in a better way.